app/component/map/stop/stop-marker-tile-layer.cjsx

```diff
@@ -0,0 +1,169 @@
+config = require('../../../config')
+React         = require 'react'
+Relay         = require 'react-relay'
+queries       = require '../../../queries'
+{VectorTile}  = require 'vector-tile'
+Protobuf      = require 'pbf'
+L             = require 'leaflet'
+BaseTileLayer = require 'react-leaflet/lib/BaseTileLayer'
+omit          = require 'lodash/object/omit'
+getSelector   = require '../../../util/get-selector'
+Popup         = require '../dynamic-popup'
+StopMarkerPopup = require './stop-marker-popup'
+provideContext = require 'fluxible-addons-react/provideContext'
+intl          = require 'react-intl'
+
+class SVGTile
+  constructor: (@coords, done, @map) ->
+    @el = document.createElementNS 'http://www.w3.org/2000/svg', 'svg'
+    @el.setAttribute "class", "leaflet-tile"
+    @el.setAttribute "viewBox", "0 0 4096 4096"
+    @el.addEventListener "click", @onMapClick
+    if @coords.z < 14
```

Comment:
Could we check this before creating  SVGTile? I mean, is there a reason to create empty SVGTile if zoom level is low?